MergedLogic-Iteration.groovy

```diff
@@ -0,0 +1,100 @@
+import hudson.model.*
+import jenkins.model.Jenkins
+import java.io.File;
+import groovy.xml.XmlUtil
+import java.io.InputStream;
+import java.io.FileInputStream
+import javax.xml.transform.stream.StreamSource
+import com.cloudbees.hudson.plugins.folder.*
+
+/*
+  The starting folder path where processing begins
+*/
+STARTINGPATH = "DevServices/Captial Markets and Finance/CMFT01/REO-VS/REOVS_RELEASE"
+
+/*
+  The folder name pattern to exclude from the recursive processing
+  NOTE: If this is empty, all the folders are skipped, so this should be set to '*' if all
+  folders should be processed
+*/
+FOLDEREXCLUSIONPATTERN = "ZRelease"
```

Comment:
`FOLDER_EXCLUSION_PATTERN `

Reply:
This has been addresses in Inclusion-Exclusion-Array.groovy